spec/features/accessibility/static_pages_spec.rb

```diff
@@ -30,6 +33,7 @@
     visit '/422'
 
     expect(page).to be_axe_clean.according_to :section508, :"best-practice", :wcag21aa
+    expect(page).to have_valid_idrefs
     expect(page).to label_required_fields
     expect(page).to be_uniquely_titled
```

Comment:
Do you think it's time to wrap up this set of assertions in a helper method?

```ruby
def verify_accessibility(page)
  expect(page).to be_axe_clean.according_to :section508, :"best-practice", :wcag21aa
  expect(page).to have_valid_idrefs
  expect(page).to label_required_fields
  expect(page).to be_uniquely_titled
end
```

Reply:
> Do you think it's time to wrap up this set of assertions in a helper method?

Updated in f602f7e + 614ecda